lib/cbus_elixir_web/templates/page/index.html.eex

```diff
@@ -31,7 +31,7 @@
         <div class="next-meeting">
             <h2 class="title myriad">Next Meeting</h2>
             <p>
-                <%= formatted_date(next_meeting()) %> at 6pm<br />
+                <%= formatted_date(@next_meeting.date) %> at 6pm<br />
```

Comment:
nil handling

Reply:
Switched to using `one!` like we were before, so this will always have a record.

Reply:
There are 100 months of meetings in the DB. I doubt there would be a nil for a meeting date.

Reply:
@croz1007 We also couldn't ever get a record with `NULL` date since this is coming from a query that compares on date. But `Meetings.next_meeting` was using plain `one` when this note was added, so that could have wound up with nil in the distant future.